class-wc-calypso-bridge.php

```diff
@@ -96,6 +97,28 @@ public function deactivate_duplicate_tiktok() {
 		}
 	}
 
+	/**
+	 * Deactivate WooCommerce Services if either Woo Shipping or Woo Tax is active.
+	 *
+	 * @since 2.3.12
```

Comment:
Change the version to x.x.x so when it's about to be released, you'll replace it with the actual one

Reply:
Is there an automated solution that will take care of this or do I have to do it manually? E.g. before merging the PR or on the `master` branch directly?

Reply:
It doesn't do it automagically, but you'd need to do it on the Release PR, once you've merged everything to master

See https://github.com/Automattic/wc-calypso-bridge/?tab=readme-ov-file#automated-deployment and a previous release PR https://github.com/Automattic/wc-calypso-bridge/pull/1435 

You'll also need to make a wpcomsh release with bumped bridge composer dependency

See https://github.com/Automattic/wpcomsh/pull/1668 and https://github.com/Automattic/wpcomsh?tab=readme-ov-file#deployment